generators/app/templates/src/utils/contextFactory/index.ts

```diff
@@ -0,0 +1,33 @@
+import { createContext, useContext, Dispatch, Reducer } from 'react';
+
+import { ActionWithType } from './interfaces';
+import withProvider from './withProvider';
+
+export default <State, Action extends ActionWithType>(
+  reducer: Reducer<State, Action>,
+  initialState: State
+) => {
+  const StateContext = createContext<State>({ ...initialState });
+  // eslint-disable-next-line @typescript-eslint/no-empty-function, no-empty-function
+  const DispatchContext = createContext<Dispatch<Action>>((() => {}) as Dispatch<Action>);
```

Comment:
`const DispatchContext = createContext<Dispatch<Action>>((jest.fn()) as Dispatch<Action>);`

Reply:
This isn't a test.